test/client/integration.test.ts

```diff
@@ -40,12 +29,13 @@ describe("Client Integration", function() {
     it("run", async function() {
         await mcl.init();
         const [signer] = await ethers.getSigners();
-        const provider = signer.provider as providers.Provider;
+        const provider = signer.provider as providers.JsonRpcProvider;
```

Comment:
In a version of these tests I tried to get two full nodes running, during which I discovered is the only `ethers` Provider that has signers is `JsonRpcProvider`.

I can revert this back to the more general interface, but felt this was worth leaving in to call out the specific provider we use for the node.